packages/odyssey-react-mui/src/ui-shell/NarrowUiShellContent.tsx

```diff
@@ -0,0 +1,404 @@
+      borderBottomWidth: odysseyDesignTokens.BorderWidthMain,
+    } as CSSProperties)),
+}));
+
+const StyledTopNavSearch = styled("div", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})<{
+  odysseyDesignTokens: DesignTokens;
+}>(({ odysseyDesignTokens }) => ({
+  paddingBlock: odysseyDesignTokens.Spacing2,
+  paddingInline: odysseyDesignTokens.Spacing3,
+}));
+
+export type NarrowUiShellContentProps = Pick<HtmlProps, "testId"> &
+  Pick<UiShellNavComponentProps, "sideNavProps" | "topNavProps"> &
+  UiShellContentProps;
+
+const NarrowUiShellContent = ({
+  appContainerElement,
+  appContainerScrollingMode,
+  hasStandardAppContentPadding = true,
+  initialVisibleSections = ["TopNav", "SideNav", "AppSwitcher"],
+  onError = console.error,
+  optionalComponents,
+  sideNavProps,
+  topNavProps,
+}: NarrowUiShellContentProps) => {
+  const odysseyDesignTokens = useOdysseyDesignTokens();
+
+  const { isContentScrolled, scrollableContentRef: appContainerRef } =
+    useScrollState(appContainerElement);
+  const sideNavContainerRef = useRef<HTMLDivElement>(null);
+  const topNavContainerRef = useRef<HTMLDivElement>(null);
+
+  const uiShellContext = useUiShellContext();
+
+  const [isLeftSideMenuOpen, setIsLeftSideMenuOpen] = useState(false);
+  const [isRightSideMenuOpen, setIsRightSideMenuOpen] = useState(false);
+
+  const closeSideMenus = useCallback(() => {
+    setIsLeftSideMenuOpen(false);
+    setIsRightSideMenuOpen(false);
+  }, []);
+
+  const toggleLeftSideMenu = useCallback(() => {
+    setIsRightSideMenuOpen(false);
+    setIsLeftSideMenuOpen((isLeftSideMenuOpen) => !isLeftSideMenuOpen);
+  }, []);
+
+  const toggleRightSideMenu = useCallback(() => {
+    setIsLeftSideMenuOpen(false);
+    setIsRightSideMenuOpen((isRightSideMenuOpen) => !isRightSideMenuOpen);
+  }, []);
+
+  const { parentContainerRef } = useAlignAppElementToContainer({
+    appContainerElement,
+    appContainerRef,
+    appContainerScrollingMode,
+    appContentPadding: hasStandardAppContentPadding ? "compact" : "none",
+    resizingRefs: [appContainerRef, sideNavContainerRef, topNavContainerRef],
+  });
+
+  return (
+    <>
+      {(isLeftSideMenuOpen || isRightSideMenuOpen) && (
+        <StyledPageOverlay
+          odysseyDesignTokens={odysseyDesignTokens}
+          onClick={closeSideMenus}
+        />
+      )}
+
+      <StyledUiShellContainer
+        odysseyDesignTokens={odysseyDesignTokens}
+        ref={parentContainerRef}
+      >
+        <StyledBannersContainer>
+          {optionalComponents?.banners}
+        </StyledBannersContainer>
+
+        {(initialVisibleSections?.includes("TopNav") || topNavProps) && (
+          <ErrorBoundary fallback={null} onError={onError}>
+            <StyledTopNav
+              isContentScrolled={isContentScrolled}
+              odysseyDesignTokens={odysseyDesignTokens}
+              topNavBackgroundColor={uiShellContext?.sideNavBackgroundColor}
```

Comment:
this needs to use `uiShellContext?.topNavBackgroundColor` instead of sideNav. this is causing the topNav to have the same background color as sidenav when it should be just neutral/white.

Reply:
The top nav color is gray, not white. I thought we wanted it to match the app color. How does it work today in Enduser? Which color does it use?